install/linux/build-deb.py

```diff
@@ -2,19 +2,15 @@
 # -*- coding: utf-8 -*-
```

Comment:
This whole file could potentially be replaced by a few commands. Maybe we should do it? :roll_eyes: 

```bash
rm deb_dist && rm dist
dpkg-buildpackage
mkdir deb_dist
mv ../*.deb deb_dist
```

Reply:
Hm, that would be a pass from me. If we want to change this script later on (as we have done already with the `--distro` argument), it's best to have a script written in Python than one in Bash / Posix shell.

Also, we'd have to `set -e` in the above Shell script, and thus make the `rm` commands not fail, etc.